components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -98,9 +98,17 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, nullable) UIView *titleView;
 
+/**
+ The font applied to the title of navigation bar.
+ Font size is enforced to 20.
+ Default is MDCTypography's titleFont.
+ */
+@property (nonatomic, strong, null_resettable) UIFont *titleFont;
```

Comment:
Also document how this property relates to `titleTextAttributes`. Notably, call out:

- `titleTextAttributes` will be deprecated in a future release and you should prefer `titleFont` instead.
- If a client sets both properties, document what the expected behavior is.

Reply:
I am sorry, everything kinda got mixed up. 
Done.